dip-0032.md

```
@@ -0,0 +1,366 @@
+| proof |ConfidentialProof| Varies | The rangeproof that the confidential amount being transferred is valid |
+
+The following data structures have been adapted from the Elements Project Transaction format:
+
+## ConfidentialAmount
+
+| Field | Required | Size | Data Type | Encoding | Notes |
+| ----- | -------- | ---- | --------- | -------- | ----- |
+| Header | Yes | 1 byte | | | A header byte of `0x08` or `0x09` indicates a blinded value encoded as a compressed elliptic curve point. With the least significant bit of the header byte denoting the least significant bit of the y-coordinate, and the remaining 32 bytes denoting the x-coordinate (big-endian). The point must be a point on the secp256k1 curve. |
+| Value |  | 32 bytes | `hex` | Big-endian | |
+
+#### ConfidentialNonce
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)